.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,6 +1,9 @@
 ---
-blank_issues_enabled: true
+blank_issues_enabled: false
 contact_links:
   - name: AEternity Community Forum
     url: https://forum.aeternity.com
-    about: Please ask and answer questions here.
+    about: Please ask and answer questions here. Most contributors can be found in the forum.
+  - name: Stack Overflow
+    url: https://stackoverflow.com/search?q=aeternity
+    about: Alternatively ask about technical issues here.
```

Comment:
Is anyone providing support at SO?

Reply:
There is an effort to establish SO as a channel. Someone from the community team is monitoring it.

Reply:
Haha, I see snippets of code that runs in a directory `~/Quviq/Aeternity` and it is posted by someone from Ukraine 🤷‍♂️

Reply:
Looks like someone is copying things (sometimes out of context) from the forum and posts it on stackoverflow. Not sure that is a great idea...

Reply:
For example https://stackoverflow.com/questions/61595805/how-to-let-hex-to-tx-obj